extensions/src/main/java/org/apache/falcon/extensions/mirroring/hdfs/HdfsMirroringExtension.java

```diff
@@ -51,6 +54,11 @@ public void validate(final Properties extensionProperties) throws FalconExceptio
     }
 
     @Override
+    public String getDescription() {
```

Comment:
In this and all the extensions below, we should not hard-code the description. Else, there will be 2 places where it will be available. You should instead do what is done in org.apache.falcon.resource.extensions.ExtensionManager#getShortDescription